Geometry/HGCalCommonData/src/HGCalWaferMask.cc

```diff
@@ -333,6 +333,44 @@ bool HGCalWaferMask::goodCell(int u, int v, int n, int type, int rotn) {
   return good;
 }
 
+int HGCalWaferMask::getRotation(int zside, int type, int rotn) {
+  if (rotn >= HGCalTypes::WaferCornerMax)
+    rotn = HGCalTypes::WaferCorner0;
+  int newrotn(rotn);
+  if ((zside < 0) && (type != HGCalTypes::WaferFull)) {
+    if (type == HGCalTypes::WaferFive) {  //WaferFive
+      static const int rot1[6] = {HGCalTypes::WaferCorner4,
```

Comment:
@bsunanda do all the `6`s here correspond to `HGCalTypes::WaferCornerMax`? if so, please replace them

Reply:
I believe they are already replaced. Can you approve this PR now?